requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
--r requirements.txt
 coverage==4.2
 django_coverage_plugin==1.3.1
 mock==1.0.1
+ipdb
```

Comment:
In terms of having a common test/dev/production, how will the requirements work? I guess we want to have setup.py still only pull in what it needs and not have deps on ipdb and therefore ipython etc. But for the docker image we will have both requirements-dev and requirements installed?